### DISCUSSION TITLE
Lesson 7: chainlink smart contracts repo not installing

### DISCUSSION THREAD

**Discussion by nazzal5448** [Cyfrin/foundry-full-course-cu#1564] [Q&A] [answered]
I am trying to install the [smartcontractkit/chainlink-brownie-contracts](https://github.com/smartcontractkit/chainlink-brownie-contracts) and I am getting the following error again and again. Although, my internet connection is working.
`$forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit`
Error:
```Installing chainlink-brownie-contracts in /mnt/d/Solidity/FundMe/lib/chainlink-brownie-contracts (url: Some("https://github.com/smartcontractkit/c
hainlink-brownie-contracts"), tag: Some("0.6.1"))                                                                                                 Cloning into '/mnt/d/Solidity/FundMe/lib/chainlink-brownie-contracts'...
remote: Enumerating objects: 33542, done.
remote: Counting objects: 100% (3143/3143), done.
remote: Compressing objects: 100% (1333/1333), done.
error: 1723 bytes of body are still expectedMiB | 412.00 KiB/s
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
fatal: clone of 'https://github.com/smartcontractkit/chainlink-brownie-contracts' into submodule path '/mnt/d/Solidity/FundMe/lib/chainlink-browni
e-contracts' failed                                                                                                                               Error: 
git submodule exited with code 128
```
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/98973132/ce9781f6-da6b-48bb-89fd-60726d362d40)

**Comment by EngrPips** [comment, marked as answer]
Hello @nazzal5448 , i have experience this so many times to know that this is a network issue. you just need to re-try whenever your network is stable and it would successfully install

**Reply by nazzal5448** [reply]
Thanks!! Yes I kep trying and it worked. Have a good day!

**Reply by EngrPips** [reply]
I am glad you fix it, keep crushing it buddy.

**Reply by hananndar** [reply]
its not working for me.

**Reply by EngrPips** [reply]
Try at a later time when your network is more stable.
